packages/client/src/components/app/filter/FilterPopover.svelte

```diff
@@ -360,7 +361,7 @@
             hide()
           }}
         >
-          Apply
+          {buttonText}
```

Comment:
You would need to ensure that this can't be empty. So add in a fallback to `Apply` to match the placeholder